src/karts/kart.cpp

```diff
@@ -470,6 +471,17 @@ float Kart::getSpeedIncreaseTimeLeft(unsigned int category) const
     return m_max_speed->getSpeedIncreaseTimeLeft(category);
 }   // getSpeedIncreaseTimeLeft
 
+// -----------------------------------------------------------------------------
+void Kart::setBoostAI(bool boosted)
```

Comment:
This method doesn't seem to ever be called? What is it for?

Reply:
Currently, it is indeed not called anywhere.

It is destined to set a given kart as using an improved AI. The expected use of this feature is in GPs : setting some karts to have a slightly better AI will allow them to have better and more consistent results compared to other AI karts, making harder to win a GP (#2777)

It is present in a patch because it is needed to make boosted_ai complete, which is introduced to determine the level of the AI item handling.